### PR TITLE
add LogPipe

### PR DESCRIPTION
**Pull request by podoliachenko**

### CODE REVIEW

File: projects/platform/src/lib/pipes/log.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+@Pipe({
+  name: 'log'
+})
+export class LogPipe implements PipeTransform {
+
+  transform(value: any, ...args: any[]): any {
```
Comment (tplk) [inline projects/platform/src/lib/pipes/log.pipe.ts:8]: Can't you just drop args in this case?

Comment (podoliachenko) [reply projects/platform/src/lib/pipes/log.pipe.ts:8]: image
I do not know.

Comment (tplk) [reply projects/platform/src/lib/pipes/log.pipe.ts:8]: `console.log` accepts and prints any number of arguments by the spec: https://console.spec.whatwg.org/#logger
But in this case your pipe doesn't really have any additional arguments.
You could use them to provide log level though.
For example:
```html
{{ value | log:'error' }}
```

```ts
transform(value: any, logLevel: 'log' | 'warn' | 'error' = 'log'): any {
  console[logLevel](value)
  return value
}
```


Technically, your current implementation can be used in a such manner:
```html
{{ '%d two %s' | log:[1, 'three'] }}
```
which will result in:
image

But it's hard to image a use case (which could be good in terms of ngxf lol).

Comment (thekiba) [reply projects/platform/src/lib/pipes/log.pipe.ts:8]: I think we need just implements others pipes: InfoPipe, WarnPipe and ErrorPipe 👌

Comment (thekiba) [reply projects/platform/src/lib/pipes/log.pipe.ts:8]: And what about console.profile, console.time and console.group?

Comment (oOlDeMoNlOo) [reply projects/platform/src/lib/pipes/log.pipe.ts]: > А как насчет console.profile, console.time и console.group?

I will do it if you see it fit.

Comment (podoliachenko) [reply projects/platform/src/lib/pipes/log.pipe.ts:8]: I will do it if you see it fit.

Comment (tplk) [reply projects/platform/src/lib/pipes/log.pipe.ts:8]: @oOlDeMoNlOo already implemented)
https://github.com/ngxf/platform/commit/1ad2fa971545d800ad612949ad02959754c9c48c

**Pull request merged**